src/modules/Dashboard/components/TakeNowModal/LabeledDropdown/LabeledUserDropdown.tsx

```diff
@@ -109,18 +179,34 @@ export const LabeledUserDropdown = ({
             fill={variables.palette[disabled ? 'on_surface_alfa38' : 'on_surface_variant']}
           />
         }
-        isOptionEqualToValue={(option, value) => value.id === option.id}
         fullWidth={true}
         value={value}
         onChange={(_e, newValue) => onChange(newValue)}
         clearOnBlur={false}
         handleHomeEndKeys
         loading={isSearching}
         loadingText={t('loadingEllipsis')}
-        noOptionsText={t('takeNowNotFound')}
+        noOptionsText={t('takeNow.modal.dropdown.notFound')}
         onInputChange={(_e, search) => debouncedSearchHandler(search)}
         {...rest}
       />
+      <RenderIf condition={shouldShowWarningMessage}>
+        <StyledFlexTopCenter
+          sx={{
+            gap: 1.6,
+            backgroundColor: variables.palette.yellow_light,
+            p: theme.spacing(0.8, 1.6),
+            mt: -1.6,
+            borderBottomLeftRadius: variables.borderRadius.sm,
+            borderBottomRightRadius: variables.borderRadius.sm,
```

Comment:
There's a lot of custom styling happening in the component definition now. I'm personally not too fussy about that, but I've seen SS folks request that we keep `sx` attributes to 2 or 3 max, and otherwise create a styled component for more advanced styling.

Reply:
I honestly don't mind it either, but I'll do some clean up